socorro/external/es/supersearch.py

```diff
@@ -540,15 +565,27 @@ def _get_histogram_agg(self, field, intervals):
             and "date_histogram"
             or "histogram"
         )
-        return A(
-            histogram_type, field=self.get_field_name(field), interval=intervals[field]
-        )
+        # NOTE(krzepka) By default ES 8.15 will return empty buckets, which creates issues in tests.
+        # Setting min_doc_count makes ES return only non-empty buckets.
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html
+        kwargs = {"min_doc_count": 1}
+        # NOTE(krzepka) "The date_histogram aggregation’s interval parameter is no longer valid."
+        # https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-8.0.html
```

Comment:
I think this is a good comment for the PR because it covers why the code changed in this PR, but not a good comment for the code. Maybe a better comment is something like this?:

```
# We want to use "calendar_interval" for date_histogram and "interval" for
# everything else.
```